content/articles/why-you-should-use-wakanda/index.md

```diff
@@ -0,0 +1,55 @@
+It helps in exposing both the REST API and server side JavaScript API. It is responsible for authentication of management sessions and users.
+
+#### Wakanda studio
+The Wakanda studio helps to develop and test apps locally together with troubleshooting your environment. In this era where everyone is craving for a code editor, Wakanda has taken a different turn in coming up with an IDE. 
+
+Wakanda's main objective is to give a quick and easy way in project scaffolding. Wakanda is the solution. The philosophy behind Wakanda studio which simplifies developers lives who have decided to work with Wakanda JavaScript framework stack is called RAD (Rapid Application Development).
+
+Do you love React? You can change the scaffolding of your project to adapt to your desire. Wakanda identifies and acknowledges the three critical parts in all applications.
+
+These crucial parts are all used in the initial scaffolding. They include the backend, web and the mobile. Studio has a code editor emanating from extension updatable, an application prototype, deploy solutions, code mirror and git integration. This studio gives the user the whole view of application.
+
+##### Application scaffolding
+It composes your app by identifying templates from the configured list. This includes Backend: typescript, JavaScript. web: Angular1, Angular4 but soon ReactJS and Aurelia are coming. Mobile: Ionic1, Ionic2.
+
+##### Environmental troubleshooting
+System and configuration problems detection tools and solutions are readily prepared for you. This will help in creating value thus choosing Wakanda JavaScript framework.
+
+### Why you should use it? 
+- It is cost effective. This is due to some of its parts being reusable modules. Its easy amalgamation between the developmental phases of application development and backend prototyping.
+- Wakanda is standardized for simplified deployment and easy maintenance.
+
+- Wakanda JavaScript framework is uniquely built. It brings together various applications which the developer needs for building, deploying, and prototyping in a single environment.
+
+- In Wakanda, cross-platform mobile and web applications can be built quickly.
+
+- Wakanda being an open-source platform is progressive so that developers make all possible choices. This means developers can take control when need be, write their code or use the available existing tools and components.
+
+- Its compatibility with all technologies make WakandaJS stand out in the market. It can integrated with databases and frameworks of all kinds. Wakanda components can be stocked by developers or rather modify them. 
+
+### Conclusion 
+To sum up, Wakanda JavaScript framework is recommendable to developers due to it's excellent and user-friendly features. The Wakanda framework also supports cross-platform applications. This means that you can use this framework to craft beautiful applications for mobile and web platforms.  
+
+Therefore, head over to Wakanda Framework and try it.
```

Comment:
Add a further reading resources section and and add links to different resources.